docs/README.md

```diff
@@ -9,6 +9,7 @@ Welcome to the NNFusion Documents!
    - [End-to-end-Test](End-to-end-Test.md)
 3. [Compile-a-Tensorflow-model-with-NNFusion](Compile-a-Tensorflow-model-with-NNFusion.md)
    - [Compile-a-model-with-kernel-tuning-enabled](Compile-a-model-with-kernel-tuning-enabled.md)
+   - [How to use NNFusion Python interface for inference/training](../src/python/example/README.md)
```

Comment:
This is for PyTorch. So, maybe the position or the higher level title needs modification.

Reply:
Yes, but what's more suitable

Reply:
Change the higher level title as "Compile a model with NNFusion", move "Compile-a-Tensorflow-model-with-NNFusion.md" to lower level? The PyTorch/ONNX frontend also needs a doc?

Reply:
Great